docs/monitoring.md

```diff
@@ -0,0 +1,143 @@
+---
+layout: default
+title: Monitoring for Migrations
+nav_order: 13
+permalink: /monitoring
+---
+
+
+# Monitoring for Migrations
+{: .no_toc }
+
+Spanner Migration Monitoring for Sharded and Non-Sharded Migrations is now available. The monitoring dahsboard can be accessed through both UI and CLI. Spanner Migration Tool uses the [**Google Cloud Monitoring Service**](https://cloud.google.com/monitoring) to create custom dashboard for each migration.
+
+<details open markdown="block">
+  <summary>
+    Table of contents
+  </summary>
+  {: .text-delta }
+1. TOC
+{:toc}
+</details>
+
+## Sharded Migration
```

Comment:
We shouldn't start with Sharded migrations. Sharded migration is a special case of a migration. 

Start with just a title like "Monitoring Migrations" and describe the non-sharded case first, then make a special distinction about this thing called "sharded" migrations and add more information about it.

Reply:
Focus more on helping the user understand what the migration dashboard is rather than defining what Harbourbridge output is. 

The harbourbridge output on the UI and CLI can come later (and lower) in the doc - probably in its own section titled something like - "How do I find my generated dashboard?" - This can be under the FAQ heading at the bottom.

Reply:
In the "How do I find my generated dashboard?" you can further say how to locate the dashboard directly on the GCP console if the HB information is not accessible.